sections/discussion/ourprocessreflections.tex

```diff
@@ -0,0 +1,58 @@
+\section{Reflections on our process}
+In \autoref{sec:our-process} we described our process and the things that we used from scrum.
+In this section we reflect on our process, the changes that we had done throughout the semester as well as what we would change for our future projects.
+
+\subsubsection{Scrum}
+Our scrum inspired process worked well with sprints, sprint planning, sprint backlog, product owner, product backlog and product increment.
+We took turns on having the role as PO.
+The PO had the task of planning the next product backlog and add a DoD onto every task in the product backlog.
+The sprint backlog and product backlog can be seen on \autoref{fig:trello-kanban}. 
+The leftmost column is the product backlog, the next four column are the sprint backlog.
+These are split into \textit{Sprint Backlog}, \textit{In progress}, \textit{In review} and \textit{Done}.
+This gives us a better overview on how far people are with their tasks.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.8\linewidth]{figures/trellopicture.PNG}
+    \caption{Our kanban board for the product and sprint backlog on Trello}
+    \label{fig:trello-kanban}
+\end{figure}
+\noindent
+When we conducted the sprint planning, we would discuss the every element in the product backlog and estimate how many story points each task should have.
+This gave everyone the same understanding on how the given tasks should be solved and eliminate most ambiguities. 
+\\
+However, later in the project when we started to develop the application and learning \textit{React}, it started to become less systematic.
+This was often due to it being difficult to estimate how long each task would take to solve.
+\\\\
+We also stopped conducting the retrospective meetings. 
+This could have helped making the project more structured as we could have discussed the problems that we faced and the reasons why we did not complete our tasks in time.
+
+\subsubsection{Formal reviews}
+The formal reviews was something that we changed in the middle of the semester, so that two people had to review a pull request together and not individually.
+Formal reviews were described in \autoref{sec:formal-reviews}.
+The first hour of every workday was spent on reviewing pull requests.
+\\\\
+This increased the quality of the pull requests, as more errors were caught when two people had to debate the code.
```

Comment:
måske noget med at det specielt er relevant for større/svære pull requests, hvor det var let at sige "det er nok fint nok"